css/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  background-color: rgb(199, 7, 199);
+}
+
+#description {
+  color: white;
+  font-size: 30px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+form {
+  font-size: 20px;
+  color: rgb(105, 248, 236);
+}
```

Comment:
I think you can do it better giving the entire page a more good-looking style. For example, it would be nice if you put some pretty border to the form :smiley: